kitchen.validate.yml

```diff
@@ -53,7 +42,19 @@ suites:
         - cron_disabled_selected_daily_and_weekly_jobs
         - openssh_installed
         - ami_cleanup_file_created
-        # TODO: enable following controls when testing install.rb recipe
-        # - efa_conflicting_packages_removed
-        # - efa_prereq_packages_installed
-        # - efa_installed
+        - c_states_kernel_configuration
```

Comment:
Nitpick. Could we rename it to past participle?

Reply:
Done